CodeMaid/Integration/Events/ShellEventListener.cs

```diff
@@ -1,160 +0,0 @@
-using Microsoft.VisualStudio;
```

Comment:
Without this class, how do you know when the user changes their Visual Studio theme (e.g. from Light to Dark) so that Spade can refresh?

Reply:
I use the built-in help class instead, see line 331 of CodeMaidPackage.cs
```csharp
    VSColorTheme.ThemeChanged += _ => ThemeManager.ApplyTheme();
```